connection/cluster/ClusterUserManager.java

```diff
@@ -73,16 +69,47 @@ public void create(String username, String password) {
         failsafeTask.runPrimaryReplica();
     }
 
+    @Override
+    public void delete(String username) {
+        ClusterClient.FailsafeTask<Void> failsafeTask = client.createFailsafeTask(
+                SYSTEM_DB,
+                parameter -> {
+                    parameter.client().stub().usersDelete(deleteReq(username));
+                    return null;
+                }
+        );
+        failsafeTask.runPrimaryReplica();
+    }
+
     @Override
     public Set<User> all() {
         ClusterClient.FailsafeTask<Set<User>> failsafeTask = client.createFailsafeTask(
                 SYSTEM_DB,
-                (parameter) ->
-                        parameter.client().stub()
-                                .usersAll(allReq()).getNamesList().stream()
-                                .map(name -> new ClusterUser(client, name))
-                                .collect(Collectors.toSet())
+                (parameter) -> parameter.client().stub().usersAll(allReq()).getUsersList().stream().map(
```

Comment:
I'd probably put a line break before one of the `.` instead of the inside the `map()` backets